supplementary_style_guide/glossary_terms_conventions/general_conventions/e.adoc

```diff
@@ -37,6 +37,22 @@
 
 *See also*:
 
+[discrete]
+[[elb]]
+==== image:images/yes.png[yes] Elastic Load Balancing (noun)
+*Description*: _Elastic Load Balancing (ELB)_ automatically distributes incoming traffic across multiple targets, such as ECS instances, and IP addresses in one or more Availability Zones. ELB supports the following load balancers:
+
+* Application Load Balancer
```

Comment:
Can I add a note to not abbreviate any of these load balancers? Creating an index for each one should not be necessary, as these component names are clearer based on the fact that they are load balancers. WDYT?

Reply:
@dfitzmau If the listed load balancers shouldn't be abbreviated, that's useful information to add. 

Not sure what you mean about abbreviating ECS if required. Can it be abbreviated or no? If not, then don't abbreviate.

Reply:
Thanks for the reply, @mportman12 . Can I add a note to the ELS index about not abbreviating the listed load balancers? Or should I create a separate index for each load balancer?

Yes, ECS is a valid initialism. Should I expand on the full term of the initialism.

Reply:
Expand ECS on first use, and add a line after the list of load balancers saying they shouldn't be abbreviated. I think that should cover it.